Source/CsDebugScript.Engine/Engine/SymbolProviders/DiaModule.cs

```diff
@@ -814,12 +814,23 @@ public BasicType GetTypeBasicType(Module module, uint typeId)
             // Partially undecorated name should be in form: "const DerivedClass::`vftable'{for `BaseClass'}"
             string partiallyUndecoratedNameStart = string.Format("const {0}{1}{{for `", codeTypeName, vftableString);
 
+            string innerCodeTypeName;
+
             if (!partiallyUndecoratedName.StartsWith(partiallyUndecoratedNameStart))
             {
-                throw new Exception("Partially undecorated name is not starting with \"" + partiallyUndecoratedNameStart + "\"");
+                partiallyUndecoratedName = partiallyUndecoratedName.Substring(0, partiallyUndecoratedName.Length - 11);
```

Comment:
just return Tuple.Create(codeType, 0); inside this if statement... It will do the same and it is more readable...